### ISSUE TITLE
Template for GUIs with seed+gotham - potential use case/example

### ISSUE THREAD

**Issue by liketechnik** [Boscop/web-view#147]
Hey there,
after posting my project on reddit, someone proposed to inform the seed contributors about it. Because you also have a section with examples/use case, I thought you could be interested in this project.
The goal of my project is to provide an 'electron like' template, that allows to create local/desktop browser GUIs with access to the local filesystem (e. g. without the usual browser restrictions). I want this for a project where I want to render latex formula in my GUI and save a lot of files.
Using seed, [gotham](https://gotham.rs/), [rust-embed](https://github.com/pyros2097/rust-embed) and [web-view](https://github.com/Boscop/web-view) the template now compiles into a single, os independent binary, that directly opens in its own window.

Let me know your thoughts on the project and thanks for an amazing library :)

P. S.: The project's repository is at: https://gitlab.com/liketechnik/local-gui-seed-gotham
P. P. S: If you think this is an appropriate example I'll be happy to open a pr for adding it (:

**Comment by khionu** [comment]
> os independent binary
I'd suggest that this be rephased, since compilation still has to be done per target. It suggests something that is impossible when it seems you mean the Bootstrap Pattern (a la the Unity and Unreal game engines).

**Comment by liketechnik** [comment]
Yes, you're right that the meaning isn't clear there. Rephrased that part.
